news/2025/2025-04-08-beatmap-spotlights-revival-spring-2025.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: post
+title: "Beatmap Spotlights Revival: Spring 2025"
+date: 2025-04-09 01:00:00 +0000
+---
+
+Haven't heard this name for a while? Check out all the major changes to Beatmap Spotlights! This time, we are also seeking fresh blood to ensure the project can run smoothly for many seasons to come!
+
+![](/wiki/shared/news/2025-04-08-beatmap-spotlights-revival-spring-2025/banner.jpg)
+
+It's been some time since we last met here indeed... let's examine what happened in the meantime, since there's certainly a lot to cover.
+
+Even though the playlists are already up and running, we invite you to read on for the story about the new Beatmap Spotlights infrastructure, and the path that led us back to where we are today!
+
+## A lengthy history lesson
+
+Heading into 2024, right after the conclusion of the Autumn 2023 season, the Beatmap Spotlights project was stopped for time being, due to an osu!(lazer) scoring rollout and infrastructure changes related to that.
+
+Some time passed, and alongside the introduction of [Daily Challenge](https://osu.ppy.sh/home/news/2024-07-25-introducing-daily-challenges) in July 2024, discussions began about how they would coexist in the future. After some deliberation, it was decided that both projects would focus on different matters, and aim for a varying player base.
+
+With that, Beatmap Spotlights could run again. However, before that could happen, various parts of Spotlights infrastructure would need to see some change.
+
+The idea was to create a new, unified system for curating the beatmaps directly on the website, along with brand-new infrastructure to handle it, including live-updating rankings and seasonal divisions, extending on [previously implemented seasons backend](https://github.com/ppy/osu-web/pull/9725).
+
+While the idea was simple, it wouldn't be easy. At this point, intense work was ahead of every person involved in Beatmap Spotlights project. Since we are finally here, it means the goals set ahead of us were successfully achieved, and so we are excited to announce major changes influencing every single inch of Beatmap Spotlights workflows.
```

Comment:
```suggestion
While the idea was simple, it wouldn't be easy. At this point, intense work was ahead for every person involved in the Beatmap Spotlights project. Since we are finally here, it means the goals set ahead of us were successfully achieved, and so we are excited to announce major changes influencing every single inch of the Beatmap Spotlights workflows.
```